CMakeLists.txt

```diff
@@ -109,6 +109,8 @@ include(SetupTribitsInstall)
 
 SET(Trilinos_MUST_FIND_ALL_TPL_LIBS_DEFAULT TRUE)
 
+SET(LINK_LIBRARIES_ONLY_TARGETS TRUE)
```

Comment:
I think setting this as a non-cache var is appropriate.  It may impact projects like Drekar that use a base Trilinos CMake project but it should be easy to for them to fix any.  I am curious to see if all of Trilinos actually configures with this turned on (since raw library names will no longer be accepted).

This should **not** cause any change in behavior for downstream Trilinos customers just pulling in Trilinos with `find_package(Trilinos ...)` or `find_package(<TrilinosPackage> ...)`.  Therefore, I am not sure it requires a release note.

Reply:
Ah good point.  This affects _developers_, not customers.  and with how TriBITS automates things it may not even affect developers all that much.